includes/Renderer/Stories/Renderer.php

```diff
@@ -37,6 +37,9 @@
 use Iterator;
 use WP_Post;
 
+// Disable reason: prevent false positives due to ReturnTypeWillChange attributes.
```

Comment:
Can you explain this one a little better?

Reply:
Because there now is `#[\ReturnTypeWillChange]` between the docblocks and the actual methods, PHPCS wrongly thinks the methods don't have any docblocks associated with them.

`#[\ReturnTypeWillChange]` annotations are needed for PHP 8.1 compatibility. See https://wiki.php.net/rfc/internal_method_return_types